### ISSUE TITLE
volume change doesnt work

### ISSUE THREAD

**Issue by jpctheman** [Minhs2/1-tick-flick-metronome#5]
default tick volume is set to 96, and default tock set to 0. however the tock is still audible on download, and they're roughly the same volume. that volume is also very quiet and unable to be turned up. id like to be able to listen to music and crank up the volume of the tick-tock of the plugin, but because it's stuck on its initial volume with no way to increase it (changing the value over 96 doesn't do anything), i can only use while playing the game in silence pretty much

**Comment by Haemylicious** [comment]
I would love to have the option to crank the volume as well, as currently having it tied to the in game sounds means it gets drowned out by creatures'/player's attacks.

**Comment by grodan-boll** [comment]
If anyone else has this issue, I would recommend comboing this plugin with the "Mute All"-plugin. It mutes all in-game sounds but allows you to have the sound-effects slider up to hear the metronome.

**Comment by jboss94** [comment]
Yeah sadly this is just way too quiet, trying to help a friend learn the inferno and he brought this plug-in up. would be better if it could be turned up probably

**Comment by swirle13** [comment]
It appears that the volume is affected by the in-game sound effects slider, but only a little. It is still audible at 0% and slightly louder at 100%, so maybe @Minhs2 is making the audio scale linearly rather than logarithmically, which is how perceived sound works.

"_Volume sliders must not be linear_. Linear volume sliders are a nuisance because human perception of loudness is not linear at all, it is _logarithmic_. That is why all audio equipment worth its name uses the dB scale to indicate volume and gain settings. For a relative amplitude level _x_, the dB value equals 20*log10(_x_). Positive dB values mean amplification, negative values attenuation. Multiplying amplitude by a certain factor means adding a certain amount of dB. To measure absolute loudness as perceived by humans, the dB(A) scale is often used, with 0 dB(A) the loudness of the most silent perceivable sound. In practice, a ‘silent’ room will be at ±30 dB(A)."
- https://www.dr-lex.be/info-stuff/volumecontrols.html

**Comment by SuperCoolDude** [comment]
Same issue, please fix it so I can have a louder metronome without having effects set to loud.

**Comment by Sefron** [comment]
There is a workaround for what you guys are complaining about.

When you mute the Sound Effects slider the tick volume goes to the level it is set to in the plugin

Though this creates an alternative problem where you can't have any sound effects...

The real solution would be decoupling the plugin sound and ingame sound completely.

**Comment by SoggyBurritoVR** [comment]
I'm working on an update to fix this issue, will submit a PR once I've finalized my updates and tested thoroughly. 

I've found a fix, just trying to get everything working as the original plugin did.

As a side effect of my fixes, now both tick and tock have their own independent volume, meaning users will now be able to set their own volumes for "Tick" & "Tock" separate from each other. 

I will be making a PR at some point on Saturday.
